packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol

```diff
@@ -86,6 +91,7 @@ contract L2StandardBridge is StandardBridge {
         uint32 _minGasLimit,
         bytes calldata _data
     ) external payable virtual {
+        // TODO: add onlyEOA check on ETH withdrawals to match L1Bridge.depositETHTo?
```

Comment:
This check should go in `withdraw` not `withdrawTo`?

Reply:
Good callout. I made that change here.